CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
 ### Changed
 - **cf-forms:** [PATCH] Fixed checkbox/radio borders on hover
+- **cf-pagination:** [PATCH] Fixed outdated `a-btn__icon-on-left`
+and `a-btn__icon-on-right` classes referenced in docs.
```

Comment:
This is nitpicky, but I think the change log should probably give either _both_ old and new class names, or just the correct ones.